### PR TITLE
kops: create a one-off job to test ARM builds

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

**Comment by justinsb** [comment]
The goal is to catch mistakes like https://github.com/kopeio/etcd-manager/pull/379

**Comment by hakman** [comment]
@justinsb a bit unrelated, but blocking k8s master tests which include the ARM ones, could your review https://github.com/kubernetes/kops/pull/10284?

File: config/jobs/kubernetes/kops/build-grid.py [hunk 1]
```diff
@@ -356,6 +356,16 @@ def generate():
...
 
+    # A one-off scenario testing arm64
+    # TODO: Would be nice to default the arm image, perhaps based on the instance type
+    build_test(force_name="scenario-arm64",
+               cloud="aws",
+               distro="u2004",
+               extra_flags=['--zones=us-east-2b',
+                            '--node-size=m6g.large',
+                            '--master-size=m6g.large',
+                            '--image=099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20201026'])
```
Comment (hakman) [inline config/jobs/kubernetes/kops/build-grid.py:367]:
```suggestion
                            '--image=099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20201026']) # pylint: disable=line-too-long
```

**Pull request merged**